MONOREPO-PLAN.md

```
@@ -0,0 +1,295 @@
+# MONOREPO-PLAN — Approach 1 execution (this worktree)
+
+**Worktree:** `/Users/Martin/Tresors/Projects/GenesisTools-monorepo-1`
+**Branch:** `feat/monorepo-1` (based on `master` @ `581f71f70f9ddfb4671628f76203be3488c06683`)
+
+Run every command with an explicit worktree path
+(`cd /Users/Martin/Tresors/Projects/GenesisTools-monorepo-1 && …` or
+`git -C … `). Never touch the main repo or sibling worktrees.
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)